scripts/utils/index.ts

```diff
@@ -97,43 +96,49 @@ export const getDeployedAddress = (initCode: string, salt: BigNumberish) => {
   );
 };
 
-export const getDeployerInstance = async (): Promise<Deployer> => {
-  const metaDeployerPrivateKey = process.env.FACTORY_DEPlOYER_PRIVATE_KEY;
-  if (!metaDeployerPrivateKey) {
-    throw new Error("FACTORY_DEPLOYER_PRIVATE_KEY not set");
-  }
-  const metaDeployer = new ethers.Wallet(
-    metaDeployerPrivateKey,
-    hardhatEthersInstance.provider
-  );
-  const deployerAddress = getContractAddress({
-    from: metaDeployer.address,
-    nonce: 0,
-  });
-
-  const provider = hardhatEthersInstance.provider;
+export const getDeployerInstance = async (
```

Comment:
do add comments throughout this function wherever applicable. 
and delete the unnecessary commented out parts...